docs/contrib/quickstart.rst

```diff
@@ -0,0 +1,123 @@
+.. _quickstart:
+
+Quick Start Guide
+=================
+
+Requirements
+------------
+
+- Java 8 or newer
+- sbt 1.5.8 or newer
+- LLVM/Clang 6.0 or newer
```

Comment:
We can mention recomended version which I belive would be 15+ (becouse it's using opaque points so it's easier to inspect LLVM IR files when contribution. 
When it comes to sbt version doesn't really matter becouse version pinned in project/build.properties would be downloaded and used anyway.

Reply:
I propose we adopt wording such as: 
"
Java 8 is the design center, some newer but not all, or even most,  newer methods may be implemented.

For sbt and LLVM/Clang, this is what is used in Scala Native Continuous Integration and is known to work. Other
versions may work, but are not guaranteed. The further an alternate version is from the
known working version, either older or newer, the less likely they are to work".

If we change the LLVM/Clang version here, which I heartily support, we should probably change
the "minimum clang check" hidden somewhere in the codebase, perhaps to a less strict "llvm-13" or
"llvm-11".

Reply:
When I am not getting kicked by the other legs of the 16 legged donkey, I am chasing one or more
JDK > 17 bugs where the initial variant of sbt may matter, and need to be greater than '1.6.0' or '1.6.2'.

I am also chasing some other cases where I suspect the initial sbt version is important.  I need to repeat
and capture what I think I saw.   More as I have proof.